src/main/java/com/sprint/mission/discodeit/service/basic/BasicChannelService.java

```diff
@@ -0,0 +1,157 @@
+package com.sprint.mission.discodeit.service.basic;
+
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.dto.*;
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.entity.ChannelType;
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+import com.sprint.mission.discodeit.repository.MessageRepository;
+import com.sprint.mission.discodeit.repository.UserRepository;
+import com.sprint.mission.discodeit.repository.ReadStatusRepository;
+import com.sprint.mission.discodeit.service.ChannelService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.time.Instant;
+import java.util.*;
+
+@Service
+@RequiredArgsConstructor
+public class BasicChannelService implements ChannelService {
+    private final ChannelRepository channelRepository;
+    private final MessageRepository messageRepository;
+    private final ReadStatusRepository readStatusRepository;
+    private final UserRepository userRepository;
+
+    @Override
+    public ChannelDto create(ChannelDto dto, List<UUID> userList) {
+        //private 신청 검토
+        Channel savedChannel = channelRepository.save(dto.type());
+        userList.forEach(userId -> readStatusRepository.save(new ReadStatusDto(null, userId, savedChannel.getId(), null, null, null)));
+        return new ChannelDto(savedChannel);
+    }
+
+    @Override
+    public ChannelDto create(ChannelDto dto) {
+        //public 신청 검토
+        Channel savedChannel = channelRepository.save(dto.name(), dto.type());
+        //readStatus 업데이트 필요
+
+        return new ChannelDto(savedChannel);
+    }
+
+    @Override
+    public ChannelFindDto find(UUID id) {
+        Channel channel = channelRepository.findById(id);
+        if(channel == null){
+            throw new IllegalStateException("채널을 찾을 수 없습니다.");
+        }
+        List<Message> messagesById = messageRepository.findMessagesById(id);
+        if(messagesById.isEmpty()) {return new ChannelFindDto(channel, null);}
+        Instant updatedAt = messagesById.stream().min(Comparator.comparing(Message::getUpdatedAt)).get().getUpdatedAt();//가장 최근의 메시지 시간 정보
```

Comment:
```suggestion
        Instant updatedAt = messagesById.stream().max(Comparator.comparing(Message::getUpdatedAt)).get().getUpdatedAt();//가장 최근의 메시지 시간 정보
```

가장 최근 메시지를 가져오려면 max가 되어야 할듯 합니다.
최근 시간의 값이 더 크답니다!